### DISCUSSION TITLE
Is it possible to provide multiple regions to the AWS provider?

### DISCUSSION THREAD

**Discussion by SantosJ1** [GoogleCloudPlatform/terraformer#1820] [Q&A]
I have noticed that when I pass multiple regions into the CLI as below:
terraformer plan aws -r vpc --profile saml --regions=us-east-1,us-west-1

Terraformer is not able to correctly "move" onto the next region and pull it's VPCs. Instead, it attempts to refresh the VPCs it pulled from the first region (which obviously fails) and then creates a plan.json file that contains no resources. Has anyone experienced this and have a work around? If not, it may be something that can be added as a feature in the future as it appears to work just fine with the GCP provider.

**Comment by slastique** [comment]
Hi @SantosJ1 I do the following:

- run `terraformer plan` with just one region
- rename plan.json to avoid overwriting with the next `terraformer plan`
- run `terraformer plan` again with another region
- merge all json files using any diff tool (it's really easy, the json file structure is simple)
- after `terraformer import` add provider configuration to `provider.tf`: you have to add provider alias and use it for resources from the second imported region, for example
